### DISCUSSION TITLE
[help] Caught Plotting Error/RuntimeError--[6872] Failed to execute script chia

### DISCUSSION THREAD

**Discussion by androidlloyd** [Chia-Network/chia-blockchain#3958] [Support]
I started 3 plots in parallel and the first of them to finish produced the following:

Final File size: 101.343 GiB
Total time = 27284.539 seconds. CPU (129.780%) Thu May  6 07:52:58 2021
Caught plotting error: remove: The process cannot access the file because it is being used by another process.: "D:\ChiaTemp\plot-k32-2021-05-06-00-18-8dcd91cd1893be66980d8098f3aaaff5c039254d4ae8fd0cb794fce5ef3e5eb1.plot.table4.tmp"
[6872] Failed to execute script chia
Traceback (most recent call last):
  File "chia\cmds\chia.py", line 81, in <module>
  File "chia\cmds\chia.py", line 77, in main
  File "click\core.py", line 829, in __call__
  File "click\core.py", line 782, in main
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1066, in invoke
  File "click\core.py", line 610, in invoke
  File "click\decorators.py", line 21, in new_func
  File "chia\cmds\plots.py", line 135, in create_cmd
  File "chia\plotting\create_plots.py", line 176, in create_plots
RuntimeError: remove: The process cannot access the file because it is being used by another process.

1) Are any of these 3 plots salvageable? If I let the other two finish, might the first one get freed up to finally move?
2) How can I identify the correct plot and manually move it out of my temp directory?
3) Will this error affect the other two plots and prevent them from finishing?
4) Any indication as to what might have caused the issue?

**Comment by infinitiLABS** [comment]
Hello
im having this issue as well, i have tried to plot on different drives and even different machines and it either spits that error or just hangs for ever. i just installed newest version 1.1.4 and would like to know how to solve this issue, is it  a software bug? this issue has me completely stuck not being able to produce plots.

did you find answer to this? thanks!

**Reply by androidlloyd** [reply]
I *think* I figured out what caused the issue. I am plotting on my workstation where I have anti-virus and backup software running in the background. I am almost positive that either a virus scan or backup was being run on my temp folder, causing the issue. I added exclusions to both the anti-virus and backup system for my temp folder and haven't seen the issue again. 

I would look at your system and try to find out what processes may be running in the background and reading/writing to files. Also, I had my plotting seize up yesterday when I ran into a bottleneck. The solution was to make sure that I'm not trying to run multiple plots in parallel so closely together that they are overlapping in phase 1. 

Hope that helps!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
